src/smb_ntlm.c

```diff
@@ -47,9 +47,8 @@
 
 #ifdef HAVE_BSD_STRING_H
 #include <bsd/string.h>
-#else
-#include "compat.h"
 #endif
+#include "compat.h"
```

Comment:
closesocket is provided with compact.h. we need this macro whether or not the system have bsd/string.h.